base/ttyhascolor.jl

```diff
@@ -0,0 +1,30 @@
+if Sys.iswindows()
+    ttyhascolor(term_type = nothing) = true
+else
+    function ttyhascolor(term_type = get(ENV, "TERM", ""))
+        startswith(term_type, "xterm") && return true
+        try
+            @static if Sys.KERNEL === :FreeBSD
+                return success(`tput AF 0`)
+            else
+                return success(`tput setaf 0`)
+            end
+        catch e
+            return false
+        end
+    end
+end
+function get_have_color()
+    global have_color
+    have_color === nothing && (have_color = ttyhascolor())
+    return have_color::Bool
+end
+in(key_value::Pair{Symbol,Bool}, ::TTY) = key_value.first === :color && key_value.second === get_have_color()
+haskey(::TTY, key::Symbol) = key === :color
+getindex(::TTY, key::Symbol) = key === :color ? get_have_color() : throw(KeyError(key))
+get(::TTY, key::Symbol, default) = key === :color ? get_have_color() : default
+<<<<<<< HEAD
+
+
+=======
+>>>>>>> 59586ded0da55b383c01b5d6e3538cd9680d79a3
```

Comment:
Remove lines 26 - 30
```suggestion
<<<<<<< HEAD


=======
>>>>>>> 59586ded0da55b383c01b5d6e3538cd9680d79a3
```